src/Sulu/Component/Webspace/Tests/Unit/Manager/PhpWebspaceCollectionDumperTest.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sulu\Component\Webspace\Tests\Unit\Manager;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Sulu\Component\Webspace\Manager\Dumper\PhpWebspaceCollectionDumper;
+use Sulu\Component\Webspace\Manager\WebspaceCollection;
+
+class PhpWebspaceCollectionDumperTest extends TestCase
+{
+    use ProphecyTrait;
```

Comment:
Think this is not required on the 2.4 branch / it does not exist on it. I will add it when merging it back to 2.5.